include/zephyr/dt-bindings/pwm/stm32_pwm.h

```diff
@@ -16,10 +16,10 @@
  * @{
  */
 /** PWM complementary output pin is enabled */
-#define PWM_STM32_COMPLEMENTARY	(1U << 8)
+#define STM32_PWM_COMPLEMENTARY	(1U << 8)
 
 /** @cond INTERNAL_HIDDEN */
-#define PWM_STM32_COMPLEMENTARY_MASK	0x100
+#define STM32_PWM_COMPLEMENTARY_MASK	0x100
```

Comment:
Let's make change non breaking
```suggestion
#define PWM_STM32_COMPLEMENTARY_MASK	__DEPRECATED_MACRO 0x100
#define STM32_PWM_COMPLEMENTARY_MASK	0x100
```

Reply:
ok